### DISCUSSION TITLE
Can I only show items before/after the current date in a listing?

### DISCUSSION THREAD

**Discussion by tim-at-topos** [quarto-dev/quarto-cli#8437] [Q&A]
### Description

I have a listing that I use to organise a seminar, where each item corresponds to a talk. I'd ideally like to split this into two listings: upcoming talks, and previous talks. I know that I can use `include` and `exclude` within a listing to glob against an author name, as explained in the docs, but is it possible to do something more advanced, namely only include posts before/after a certain date?

**Comment by mcanouil** [comment]
If you write a pre-render script (or a script you run before `quarto render`), that checks the date and creates two YAML contents files you pass to the two listings.

See:
- https://quarto.org/docs/projects/scripts.html#pre-and-post-render
- https://quarto.org/docs/websites/website-listings.html#yaml-listing-content

Somewhat related:
- #3114
